### DISCUSSION TITLE
Çıktı aynı olmasına rağmen hatalı diyor sistemsel arıza mı?

### DISCUSSION THREAD

**Discussion by atlasmarsaren** [19/classroom#17] [Anonim] [answered]
![650168e8-2453-4ad2-921b-b6dd486c4885](https://user-images.githubusercontent.com/92267800/157966721-03f68fd2-7753-4117-8b47-13c518a49383.jpg)

**Comment by roktas** [comment, marked as answer]
Hayır çıktılar aynı değil, yazdığın kodu dikkatli incele, "Actual output"ta sonda (ekran görüntüsünde görünmeyen) boşluk karakterleri var.

Bir de...  Lütfen bu tür durumlarda hemen "sistemsel arıza mı var?" diye atlamayın.  Bu ödevler uğraşmanız için var.  Ödevin teslimine daha günler varken, ilk saatlerde bu tür hata kayıtları açmayın.  Bir uğraş bakalım 3-5 gün, olmadı sorarsın.

**Comment by roktas** [comment]
[İlave bilgi](https://roktas.dev/classroom/help.html#lets-grad-komutunun-kars%C4%B1last%C4%B1rma-c%C4%B1kt%C4%B1lar%C4%B1n%C4%B1-nas%C4%B1l-yorumlamal%C4%B1y%C4%B1m)
